toqito/matrix_props/is_absolutely_k_incoherent.py

```diff
@@ -0,0 +1,115 @@
+"""Checks if the matrix is absolutely $k$-incoherent."""
+
+import cvxpy as cp
+import numpy as np
+
+from toqito.matrix_props import is_positive_semidefinite, is_square
+
+
+def is_absolutely_k_incoherent(mat: np.ndarray, k: int, tol: float = 1e-15) -> bool:
+    r"""Determine whether a quantum state is absolutely k-incoherent :cite:`Johnston_2022_Absolutely`.
+
+    This function checks if the provided density matrix is absolutely k-incoherent based on the criteria introduced in
```

Comment:
Minor nit: we should summarize the criteria in this docstring

Reply:
Done here as well 55a368f0d9211be1cca7bdc1e8df80986fa87ef7